src/components/Home/index.tsx

```diff
@@ -48,7 +85,10 @@ export function Home({ user }: IUser) {
 
   return (
     <div>
-      Welcome {user.name}! <a href="/api/auth/logout">Logout</a>
```

Comment:
Como vc tem um onClick, utilize um button em vez da ancora, e para redirecionar sempre use formas do próprio next para nao gerar reloads.

Para ancora [import Link from 'next/link']
<Link href='page'>

Para botao [import {useRouter} from 'next/router']
<button onClick={() => router.push('/about')}>
  Click here to read more
</button>

Reply:
Beleza, por enquanto esta como ancora so para teste, vou mudar para button e o redirecionamento quando fizer o menu nav bar